crates/aiken/src/cmd/new.rs

```diff
@@ -165,7 +165,7 @@ fn gitignore(root: &Path) -> miette::Result<()> {
         indoc! {
             r#"
                 # Aiken compilation artifacts
-                assets/
+                artifacts/
```

Comment:
Cool and then now we can confidently .gitignore this?

Reply:
Yup.